.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "monthly"
```

The description says checks are intended to be done monthly. Maybe update the comment accordingly?
```suggestion
      # Check for updates to GitHub Actions every month
      interval: "monthly"
```